tools/gfx.c

```diff
@@ -155,7 +163,7 @@ void remove_duplicates(struct Graphic *graphic) {
 }
 
 bool flip_exists(uint8_t *tile, uint8_t *tiles, int tile_size, int num_tiles, bool xflip, bool yflip) {
-	uint8_t *flip = calloc(tile_size, 1);
+	uint8_t *flip = alloca(tile_size);
```

Comment:
Just use the array syntax.

Reply:
Wasn't sure if the codebase was targeting C99 or not, I'll do that.